.github/workflows/sub-deploy-integration-tests-gcp.yml

```diff
@@ -659,14 +659,30 @@ jobs:
         run: |
           MINIMUM_UPDATE_HEIGHT=$((ORIGINAL_HEIGHT+CACHED_STATE_UPDATE_LIMIT))
           if [[ -z "$UPDATE_SUFFIX" ]] || [[ "$SYNC_HEIGHT" -gt "$MINIMUM_UPDATE_HEIGHT" ]] || [[ "${{ inputs.force_save_to_disk }}" == "true" ]]; then
+
+             # Use RUNNING_DB_VERSION for image naming (more reliable than STATE_VERSION)
+             # Extract just the major version number for the image name
+             IMAGE_VERSION_FOR_NAME=${RUNNING_DB_VERSION#v}  # Remove v prefix
```

Comment:
Could we exclude the db version altogether and bring it back if and when we have another major db format upgrade that is not restorable from the previous one? There shouldn't be any caches from before v25 (the first reusable major version), following versions are restorable, and I don't foresee any major version bumps to the db format that would really need to sync from scratch.